src/docs/development/accessibility-and-localization/accessibility.md

```diff
@@ -53,6 +53,14 @@ we recommend using automated accessibility scanners:
     1. In the Accessibility Inspector,
        select **Audit** in the toolbar, and then
        select **Run Audit** to get a report of potential issues
+ 
+  * For web:
+    1. Turn on Screen Reader for your OS
+    2. Navigate to the Enable accessibility toggle using the Screen Reader to build the semantics tree. 
+    You can skip this step if you programmatically auto-enable accessibility for your app using this API: `RendererBinding.instance.setSemanticsEnabled(true)`
```

Comment:
The instructions above are for developers to inspect the accessibility tree, but the web instructions read like a user guide for accessibility user to use the app. For developers, we could recommend using Chrome DevTools (and equivalents in other browsers) to inspect the HTML tree containing the ARIA attributes generated by Flutter. In Chrome (and probably new Edge) the "Elements" tab has "Accessibility" sub-tab that can be used to inspect the data exported to the semantics tree.